docs/src/howto/install/snap.md

```diff
@@ -0,0 +1,69 @@
+# Install Canonical Kubernetes from a snap
+
+Canonical Kubernetes is packaged as a [snap], available from the 
+snap store for all supported platforms.
+
+## What you'll need
+
+This guide assumes the following:
+
+- You are installing on Ubuntu 22.04 or later, **or** another OS which supports
+  snap packages (see [snapd support])
+- You have root or sudo access to the machine
+- You have an internet connection
+- The target machine has sufficient memory and disk space. To accommodate
+  workloads, we recommend a system with at least 20G of disk space and 4G of
```

Comment:
4G / 20G is fine for a basic install, but I think if we're gonna say *recommend*, we should bump those up. If something like 8G feels too heavy, perhaps emphasize the **at least** part and wrap it in a `<blink>` tag or something.

FWIW, we're upping the charmed k8s k-c-p recommended specs to 8G / 64G in https://github.com/charmed-kubernetes/bundle/pull/892.